build

```diff
@@ -7,7 +7,7 @@ TARGET="$2" # Example: riscv64-linux-gnu
 MCPU="$3" # Examples: `baseline`, `native`, `generic+v7a`, or `arm1176jzf_s`
 
 ROOTDIR="$(pwd)"
-ZIG_VERSION="0.8.0-dev.1813+88d40fc00"
+ZIG_VERSION="0.8.0-dev.1702+f67e75621"
```

Comment:
where did this 1702 number come from? it should be `2167` or thereabouts.

anyway my bad I didn't see this PR and I did it in master branch!